.azure_pipelines/performance_check/configs/bert_gpu.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
How is this different from examples/bert/bert_cuda_gpu.json ?

Reply:
This is same as bert_cuda_gpu in our example.